src/parser/xasl_generation.c

```diff
@@ -81,6 +81,43 @@ extern void qo_plan_lite_print (QO_PLAN * plan, FILE * f, int howfar);
 /* maximum number of functions that can be optimized */
 #define ANALYTIC_OPT_MAX_FUNCTIONS              32
 
+typedef struct hashable HASHABLE;
+struct hashable
+{
+  bool is_PRIOR;
+  bool NAME_without_prior;
```

Comment:
naming; 
- Only lowercase and underscore(`_`) is recommended for struct data member variables. 
- Because NAME_without_prior is also boolean, it has to start with `is_` prefix.

Reply:
I fixed it.